doc/src/manual/profile.md

```diff
@@ -194,14 +208,14 @@ be very useful, but sometimes you want to start fresh; you can do so with [`Prof
 [`Profile.print()`](@ref) has more options than we've described so far. Let's see the full declaration:
 
 ```julia
-function print(io::IO = STDOUT, data = fetch(); format = :tree, C = false, combine = true, cols = tty_cols(), maxdepth = typemax(Int), sortedby = :filefuncline)
+function print(io::IO = STDOUT, data = fetch(); kwargs...)
 ```
 
-Let's discuss these arguments in order:
+Let's first discuss the two first arguments and then later all the different possible keyword arguments:
```

Comment:
Perhaps "Let's first discuss the two positional arguments, and later the keyword arguments:"?